controllers/applicationsnapshotenvironmentbinding_controller.go

```diff
@@ -158,6 +158,7 @@ func (r *SnapshotEnvironmentBindingReconciler) Reconcile(ctx context.Context, re
 	var tempDir string
 	clone := true
 
+	appSnapshotEnvBinding.Status.Components = []appstudiov1alpha1.BindingComponentStatus{}
```

Comment:
so do we calculate the Status.Components every reconcile?

Reply:
Yes, see https://github.com/redhat-appstudio/application-service/pull/358/files#diff-b6bdb0d620155ee1d570059beb3dc2335bee5fb63a6e8ab12ace2b705b51a00cR364